front/issue.form.php

```diff
@@ -48,14 +48,22 @@
    $issue->saveAnswers($_POST);
    Html::back();
 } else {
+   $issue->getFromDB((int) $_REQUEST['id']);
+
+   // Accessing a form from a tech profile, redirect to ticket page if possible
+   if (Session::getCurrentInterface() == 'central') {
+      $id = $issue->fields['original_id'];
```

Comment:
**original_id** and **sub_itemtype** were renamed respectively to **items_id** and **itemtype**

Also, for maintenance readability, I think this redirection should be located in setup.php, after the enlightened if() statement.

![image](https://user-images.githubusercontent.com/14139801/127996230-1933c949-e7b8-45b0-9f91-f31f4fb779c6.png)

Reply:
> original_id and sub_itemtype were renamed respectively to items_id and itemtype

Good catch, I tested on the previous version and assumed it didn't changed in 2.12... should have tested on this version too :)
Fixed fa7ee21b.

For the redirection I can change it if you want but it seemed simpler for me to have it in the file directly targeted by the link.
-> Make it easier for a developer to finds it as you can directly check the file you were trying to access (issue.form.php) instead of searching for another unrelated file.

It is different that the others redirect done in `setup.php` that target core files and thus can't be done directly in them since we are in a plugin.

Reply:
OK, this is not a big concern anyway, but let's wait for the merge of #2321 to rebase this one and see if there is a conflict.